test/reload_config.bats

```diff
@@ -33,31 +33,22 @@ function expect_log_failure() {
 	ps --pid "$CRIO_PID" &> /dev/null
 }
 
-@test "reload config should succeed with 'log_level'" {
+@test "log_level set via command line remains after reload" {
```

Comment:
Is there a plan to add these tests in a separate BATS file to ensure their coverage?

Reply:
I meant the ones you removed.

Reply:
Sorry, forgot to include that file. Added now!